lib/gemojione.rb

```diff
@@ -63,7 +72,12 @@ def self.image_url_for_unicode_moji(moji)
   end
 
   def self.image_tag_for_moji(moji)
-    %Q{<img alt="#{moji}" class="emoji" src="#{ image_url_for_unicode_moji(moji) }"#{ default_size ? ' style="width: '+default_size+';"' : '' }>}
+    if use_sprite
```

Comment:
There seems to be no test for this case, just the else case that was implemented